presto-main-base/src/main/java/com/facebook/presto/sql/planner/sanity/PlanChecker.java

```diff
@@ -73,7 +73,8 @@ public PlanChecker(FeaturesConfig featuresConfig, boolean noExchange, PlanChecke
                         new VerifyNoIntermediateFormExpression(),
                         new VerifyProjectionLocality(),
                         new DynamicFiltersChecker(),
-                        new WarnOnScanWithoutPartitionPredicate(featuresConfig));
+                        new WarnOnScanWithoutPartitionPredicate(featuresConfig),
+                        new CallDistributedProcedureValidator());
```

Comment:
@hantangwangd it would be nice to have plan tests, like `TestHashGenerationOptimizer`, that show the type of plan that gets generated by a distributed procedure.